crates/uv-client/src/base_client.rs

```diff
@@ -166,6 +167,25 @@ impl<'a> BaseClientBuilder<'a> {
         self
     }
 
+    /// Read the retry count from [`EnvVars::UV_HTTP_RETRIES`] if set, otherwise, make no change.
+    ///
+    /// Errors when [`EnvVars::UV_HTTP_RETRIES`] is not a valid u32.
+    pub fn retries_from_env(self) -> anyhow::Result<Self> {
+        // TODO(zanieb): We should probably parse this in another layer, but there's not a natural
+        // fit for it right now
+        if let Some(value) = env::var_os(EnvVars::UV_HTTP_RETRIES) {
+            Ok(self.retries(
+                value
+                    .to_string_lossy()
```

Comment:
Can probably be `to_str` since if it's non-UTF-8 it won't parse, right?

Reply:
Then I need to write an error case for when that's returns `None`, it seemed easier to just let it hit the parse error.